src/proxy.c

```diff
@@ -477,7 +475,8 @@ gboolean is_incoming_eavesdropping (DBusMessage *msg)
                                DBUS_TYPE_STRING,
                                &msg_arguments);
 
-        if (strstr(msg_arguments, "eavesdrop=true") != NULL)
+        if (strstr(msg_arguments, "eavesdrop=true") != NULL ||
+            strstr(msg_arguments, "eavesdrop='true'") != NULL)
```

Comment:
Do you know if `eavesdrop=true` is valid? It looks like https://dbus.freedesktop.org/doc/dbus-specification.html only mentions `eavesdrop='true'` but perhaps I am missing some "escape rule".

But also, no harm in leaving the string without `'` in it... I guess.

Reply:
According to match rule parsing code, function doing that: https://github.com/freedesktop/dbus/blob/master/bus/signals.c, both tokens which are to the right side of `=` are valid: `true` and `'true'`

Reply:
You are right. https://github.com/freedesktop/dbus/blob/03158095968de8a9542b7ce448d5e9b36f10d827/bus/signals.c#L591 removes `\'` so both `strstrs()` should be kept.